packages/ui/src/components/DataStory/events/eventManager.ts

```diff
@@ -35,8 +36,10 @@ export const eventManager = new EventManager();
  * @param {EventHandler} handler
  */
 export const useDataStoryEvent = (handler: EventHandler) => {
+  const handlerRef = useLatest(handler);
```

Comment:
Does this change mean we can optionally tie a event to an Element? Can you give a little bit of background here please.